TADashboard/src/main/java/page_objects/DataProfilesPage.java

```diff
@@ -1,23 +1,23 @@
 package page_objects;
 
 import com.logigear.control.common.imp.CheckBox;
+import com.logigear.control.common.imp.Label;
 import com.logigear.control.common.imp.Link;
 import utils.common.Constants;
 
 public class DataProfilesPage extends BasePage {
-    private CheckBox dynamicCheckbox = new CheckBox("xpath=//table[@class='GridView']/tbody/tr/td[.='%s']/preceding-sibling::td/input");
-    private Link dynamicDataProfile = new Link("xpath=//table[@class='GridView']/tbody/tr/td[.='%s']");
-    private Link dynamicPresetAction = new Link("xpath=//table[@class='GridView']/tbody/tr/td[.='%s']/following-sibling::td[@class='center']/a/parent::td");
+    private CheckBox dynamicDataProfileCheckbox = new CheckBox("xpath=//table[@class='GridView']/tbody/tr/td[.='%s']/preceding-sibling::td/input");
+    private Label dynamicDataProfile = new Label("xpath=//table[@class='GridView']/tbody/tr/td[.='%s']");
+    private Link dynamicDataProfileAction = new Link("xpath=//table[@class='GridView']/tbody/tr/td[.='%s']/following-sibling::td[@class='center']/a/parent::td");
     private Link lnkAddNew = new Link("css=div[class='panel_tag2'] a[href='profile.jsp?action=create']");
 
     public String getTextOfAction(String profileName) {
-        dynamicPresetAction.setDynamicValue(profileName.replace(" ", " "));
-        return dynamicPresetAction.getText();
+        dynamicDataProfileAction.setDynamicValue(profileName.replace(" ", " "));
+        return dynamicDataProfileAction.getText();
     }
 
     public void clickOnDataProfile(String profileName) {
-        dynamicDataProfile.setDynamicValue(profileName.replace(" ", " "));
-        dynamicDataProfile.click();
+        dynamicDataProfile.getChildElement(String.format("//table[@class='GridView']/tbody/tr/td[.='%s']/a", profileName.replace(" ", " "))).click();
```

Comment:
this is not the child of data profile,

Reply:
Yeah, I got it

Reply:
Please check again. I think it is right now